### ISSUE TITLE
Running XPL on Windows

### ISSUE THREAD

**Issue by banana-cultivator** [schachmett/xpl#2]
Hi. I have an issue regarding opening this program. I tried to run it in the Anaconda prompt and without success. But I tried to install it, and there was no problem, tried to update it, again without error, but when I tried to open it, with >>>pip -m xpl, it shows error, like xpl not recognized. Any advice regarding this? Thank you kindly.

**Comment by schachmett** [comment]
Hi and sorry for the late reply. I will continue working on this once I have more time. Have you tried 
```
$ python3 -m xpl
```
directly from the command line instead of invoking `pip` from the python interpreter?
Also, which operating system are you using? I only test on Linux but I know this can be made to work on OS X, too. On Windows however I have tried and so far failed to run this app. On Linux, installing via pip should also install a `.desktop` file so you can just launch this from your app drawer.

**Comment by banana-cultivator** [comment]
Yes, I also tried this suggested solution. Still error (python3 is not recognized.....). I'm using Windows 7. So I got curious if it's working in Linux why it can't on windows too?  Anyway, thanks. I will continue to search for the solution to this interesting issue.

**Comment by schachmett** [comment]
`python3` is the command for the Linux shell (terminal window) to run Python 3, so this won't work in a python interactive session (the one with `>>>` in the front). In Windows I don't exactly know how to do this, but it's something like running cmd.exe and then executing `\path\to\python3\python -m xpl`, where the `\path\to\python3` is the path on your machine where Python 3.5 or higher is installed.

However the main issue with Windows is to install GTK3, the GUI toolkit that I used here and that XPL depends on. I never got it to run on Windows 10 or at least Python never found the GTK files. On many Linux distributions, GTK is preinstalled and on the others it is easy to install, that is why it does work there. If you know a method to install GTK3 on windows, I could try again to run XPL on there.

edit: I just remembered that in the README I already wrote that I could not get it to work on Windows. There are some links from when I tried to, maybe they lead you in the right direction.
